### PR TITLE
Added A Method for Detection of Small Flying Objects in UAV Videos

### PR DESCRIPTION
**Pull request by tze-min**
Hey! Have added the data and models to DagsHub based on #44. Let me know if you guys need any changes made.

### CODE REVIEW

File: Detection of Small Flying Objects in UAV Videos/README.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# Detection of Small Flying Objects in UAV Videos
```
Comment (deanp70) [inline Detection of Small Flying Objects in UAV Videos/README.md:1]: Missing a link to the DagsHub repo

File: Detection of Small Flying Objects in UAV Videos/README.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+This repository contains the code used in implementation of the paper Vladan Stojnić, Vladimir Risojević, Mario Muštra, Vedran Jovanović, Janja Filipi, Nikola Kezić, and Zdenka Babić, [\"A Method for Detection of Small Moving Objects in UAV Videos\"](https://www.mdpi.com/2072-4292/13/4/653), published in Remote Sensing.
+
+Dataset with used videos can be obtained at https://doi.org/10.5281/zenodo.4400650 .
```
Comment (deanp70) [inline Detection of Small Flying Objects in UAV Videos/README.md:5]: You can replace the URL to link to the DagsHub hosted dataset

**Review by deanp70 — CHANGES_REQUESTED** [review]
Hi, thanks for the contribution! I made a few comments in the code, but also, could you please add something referring to the models being tracked? I think this isn't mentioned in the readme, and might be relevant to many community members.

Thanks! After the comments are fixed we can merge this in :)

File: Detection of Small Flying Objects in UAV Videos/README.md (hunk 1 above)
Comment (tze-min) [reply Detection of Small Flying Objects in UAV Videos/README.md:1]: The link to the DagsHub repo is in line 11 of the readme, but I can shift it up later if you'd like. Re: the other changes, will do tomorrow!

**Comment by nirbarazida** [comment]
Approved 🎉 
Thanks for the contribution @tze-min

**Pull request merged**